config/rbac/role.yaml

```diff
@@ -143,3 +146,28 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - get
+  - create
+  - list
+  - delete
+  - update
+  - patch
+  - watch
+- apiGroups:
+  - rbac.authorization.k8s.io
```

Comment:
I worry about adding these permissions to the operator, is there any other way of achieving this?

Reply:
@shawn-hurley we need these permissions so that the operator creates the appropriate role/rolebindings needed for the prometheus operator to scrape the metrics from our ns.

Reply:
Hmm, isn't that already done by OLM when installing if you have the correct annotation?

Reply:
@shawn-hurley PTAL I have removed the prometheus role/rolebinding creation by OADP operator.